subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1190,6 +1190,16 @@ static int cmd_wifi_reg_domain(const struct shell *sh, size_t argc,
 	if (regd.oper == WIFI_MGMT_GET) {
 		shell_fprintf(sh, SHELL_NORMAL, "Wi-Fi Regulatory domain is: %c%c\n",
 			regd.country_code[0], regd.country_code[1]);
+		shell_fprintf(sh, SHELL_NORMAL,
+			"<center freq>\t<supported(y/n)>\t<max power(dBm)>\t<passive scan supported(y/n)>\t<dfs supported(y/n)>\n");
```

Comment:
shouldn't `supported` be first?

Reply:
`"supported(y/n)>\t<center freq>\t<max power(dBm)>\t<passive only(y/n)>\t<DFS supported(y/n)>\n"`

Reply:
I think channel number(frequency) should come first, as it's easier to search. User will search frequency first then supported otr not.

Reply:
Should we add `channel number supported center frequency`?